onward/app/controllers/package.scala

```diff
@@ -0,0 +1,13 @@
+import slices.FixedContainers
+
+package object controllers {
```

Comment:
(Though I can understand why you don't like package objects themselves. I could just stick the function inside another explicitly named object.)